src/js/tabs/osd.js

```diff
@@ -1868,13 +1876,13 @@ OSD.updateDisplaySize = function() {
     }
     // compute the size
     OSD.data.displaySize = {
-        x: FONT.constants.SIZES.LINE,
+        x: OSD.constants.VIDEO_COLS[videoType],
         y: OSD.constants.VIDEO_LINES[videoType],
         total: null,
     };
     // Adjust css background grid
     const previewLayoutElement = $(".tab-osd .display-layout");
-    videoType === 'PAL' ? previewLayoutElement.addClass('video-pal').removeClass('video-ntsc') : previewLayoutElement.addClass('video-ntsc').removeClass('video-pal');
+    videoType === 'PAL' || videoType == 'HD' ? previewLayoutElement.addClass('video-pal').removeClass('video-ntsc') : previewLayoutElement.addClass('video-ntsc').removeClass('video-pal');
```

Comment:
Rather would use a new `video-hd` (or maybe `video-hdzero`, `video-walksnail` class) instead of re-using PAL (class).

Reply:
true. don't actually need the class at all any more - it was used to swap the image backgrounds out...

Reply:
removed now.